lib/gooddata/lcm/actions/synchronize_ldm.rb

```diff
@@ -89,6 +89,11 @@ def sync_segment_ldm(params, segment_info)
             maql_diff_params << :excludeFactRule if exclude_fact_rule
             maql_diff_params << :includeDeprecated if include_deprecated
             maql_diff = previous_master.maql_diff(blueprint: blueprint, params: maql_diff_params)
+            chunks = maql_diff['projectModelDiff']['updateScripts']
+            GoodData.logger.info "Synchronize LDM to clients will not proceed in mode \
+'#{params[:synchronize_ldm].downcase}' due to no change in new released LDM. \
```

Comment:
due to no LDM changes in the new master project.

Reply:
updated